src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        int numericBalls = 3;
+        for (int i = 0; i < numericBalls; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
```suggestion
            System.out.println(ball);
```

`toString()` method will be called "under the hood"